src/main/java/seedu/address/logic/commands/SetRsvpCommand.java

```diff
@@ -16,12 +16,14 @@ public class SetRsvpCommand extends UndoableCommand {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Sets the RSVP status of a guest. "
-            + "Parameters: INDEX (must be a positive integer) "
-            + "STATUS (1 - Coming, 2 - Not Coming, 3 - Pending)\n";
+            + "Parameters: INDEX (must be a positive integer and within number of guests in address book) "
+            + "STATUS (s/1 - Coming, s/2 - Not Coming, s/3 - Pending)\n"
+            + "Ex. setrsvp 1 s/1";
 
     public static final String MESSAGE_SET_SUCCESS = "RSVP status updated for guest: ";
     public static final String MESSAGE_INVALID_INDEX = "ERROR: Please enter a valid index (from 1 to ";
-    public static final String MESSAGE_INVALID_ACTION = "ERROR: Please enter a valid action (1, 2, or 3)";
+    public static final String MESSAGE_INVALID_ACTION = "ERROR: Please enter a valid action "
```

Comment:
I think the error message should be named to be clearer. i.e. MESSAGE_INVALID_STATUS and ERROR: Please enter a valid RSVP status

Reply:
ok this can be changed